src/components/NewLexemeForm.vue

```diff
@@ -46,20 +46,26 @@ export default {
 		<lexical-category-input
 			v-model="lexicalCategory"
 		/>
-		<wikit-button
-			class="form-button-submit"
-			type="progressive"
-			variant="primary"
-		>
-			{{ $messages.get( 'wikibaselexeme-newlexeme-submit' ) }}
-		</wikit-button>
+		<div>
```

Comment:
I think we should consider this the footer of the form, or at least a section of it.

Reply:
Sure. Is there anything you want me to change for that?

Reply:
Sorry, I didn't make the actionable clear. Yeas, I think we should use an HTML `<footer>` element.

Reply:
Mh, not sure if I would go that far, to be honest. This is still a central component of the form that just happens to appear at the form's end.

A `<footer>` seems to usually contain a different class of content. From the [spec](https://html.spec.whatwg.org/multipage/sections.html#the-footer-element):

> A footer typically contains information about its section such as who wrote it, links to related documents, copyright data, and the like.

The submit button of a form doesn't feel like it belongs into that list.

Reply:
Sure, just a suggestion :)